lib/jekyll-redirect-from/generator.rb

```diff
@@ -22,11 +22,15 @@ def generate(site)
         generate_redirect_to(doc)
       end
 
-      generate_redirects_json
+      generate_redirects_json if generate_redirects_json?
     end
 
     private
 
+    def option(key)
```

Comment:
Lets do away with this method for now. We can introduce it in the future if necessary.